pkg/reporting/pullRequestReport.go

```diff
@@ -0,0 +1,421 @@
+			}
+		}
+
+		// VulnerabilitiesTable sort & copy
+		for key := range vulnerabilities {
+			item := vulnerabilities[key]
+			sort.Slice(item, func(i, j int) bool {
+				return scoreLogicSort(item[i].VulnerabilityScore, item[j].VulnerabilityScore)
+			})
+			scanReport.VulnerabilitiesTable = append(scanReport.VulnerabilitiesTable, Vulnerabilities{
+				PolicyViolationName: key,
+				Values:              item,
+			})
+		}
+		sort.Slice(scanReport.VulnerabilitiesTable, func(i, j int) bool {
+			return scanReport.VulnerabilitiesTable[i].PolicyViolationName < scanReport.VulnerabilitiesTable[j].PolicyViolationName
+		})
+
+		// LicensesTable sort & copy
+		for key := range licenses {
+			item := licenses[key]
+			sort.Slice(item, func(i, j int) bool {
+				if item[i].LicenseName < item[j].LicenseName {
+					return true
+				}
+				if item[i].LicenseName > item[j].LicenseName {
+					return false
+				}
+				return item[i].ComponentName < item[j].ComponentName
+			})
+			scanReport.LicensesTable = append(scanReport.LicensesTable, Licenses{
+				PolicyViolationName: key,
+				Values:              item,
+			})
+		}
+		sort.Slice(scanReport.LicensesTable, func(i, j int) bool {
+			return scanReport.LicensesTable[i].PolicyViolationName < scanReport.LicensesTable[j].PolicyViolationName
+		})
+
+		// OtherViolationsTable sort & copy
+		for key := range otherViolations {
+			item := otherViolations[key]
+			sort.Slice(item, func(i, j int) bool {
+				return item[i].ComponentName < item[j].ComponentName
+			})
+			scanReport.OtherViolationsTable = append(scanReport.OtherViolationsTable, OtherViolations{
+				PolicyViolationName: key,
+				Values:              item,
+			})
+		}
+		sort.Slice(scanReport.OtherViolationsTable, func(i, j int) bool {
+			return scanReport.OtherViolationsTable[i].PolicyViolationName < scanReport.OtherViolationsTable[j].PolicyViolationName
+		})
+	}
+
+	tmpl, err := template.New("report").Parse(rapidReportMdTemplate)
+	if err != nil {
+		return nil, errors.New("failed to create Markdown report template err:" + err.Error())
+	}
+	buf := new(bytes.Buffer)
+	err = tmpl.Execute(buf, scanReport)
+	if err != nil {
+		return nil, errors.New("failed to create Markdown report template err:" + err.Error())
+	}
+
+	return buf, nil
+}
+
+// getScore extracts score or severity from error message
+func getScore(message, key string) string {
+	indx := strings.Index(message, key)
+	if indx == -1 {
+		return ""
+	}
+	var result string
+	var notFirstSpace bool
+	for _, s := range message[indx+len(key):] {
+		if s == ' ' && notFirstSpace {
+			break
+		}
+		notFirstSpace = true
+		result = result + string(s)
+	}
+	return strings.Trim(result, " ")
+}
+
+// scoreLogicSort sorts two scores
+func scoreLogicSort(iStr, jStr string) bool {
+	if strings.Contains(iStr, "10.0") {
+		return true
+	} else if strings.Contains(jStr, "10.0") {
+		return false
+	}
+	if iStr > jStr {
+		return true
+	} else if iStr < jStr {
+		return false
+	}
+	return iStr < jStr
```

Comment:
I think it could be simplified

```suggestion
	if iStr >= jStr {
		return true
	}
	
	return false
```